IdentityCore/xcconfig/identitycore__idlib__ios.xcconfig

```diff
@@ -1,2 +1,25 @@
 #include "identitycore__lib__common__ios.xcconfig"
 #include "identitycore__idlib__common.xcconfig"
+
+MS_SIM_ARCHS = x86_64 i386
+PRE_XCODE_10_1_ARCHS = armv7 armv7s arm64
```

Comment:
two questions:
1. Can we use standard archs macro here?
2. Why do we need to exclude arm64e explicitly? This again seems a bit error prone, since as e.g. iphoneos13.0 becomes available, we'll likely forget to update this.

Reply:
Good point. I actually copied it from App Center configuration and App Center supports Xcode 8/9. Since we are dropping Xcode 8/9 quite soon, I can clean this up.

Reply:
Yes this can be cleaned up. We had to exclude arm64e when we were compatible with XCode 8/9/10.0.
About adding `armv7s`, it's true that `armv7` supports covers `armv7s` though it won't contains the optimizations. Some customers want these optimizations and since it's cheap we just add it. That gives the final decision on adding armv7s support to the customer.